package-lock.json

```diff
@@ -41,6 +41,7 @@
       "dev": true,
       "hasInstallScript": true,
       "devDependencies": {
+        "@azure-tools/lint-diff": "file:lint-diff",
```

Comment:
TODO: Ensure `npm i` is executed properly before updating global package-lock.json